inputs/centauri/hyperspace.nix

```diff
@@ -1,6 +1,11 @@
 { self, ... }: {
   perSystem = { config, self', inputs', pkgs, system, crane, ... }: {
     packages = rec {
+      hyperspace-dali-container = pkgs.dockerTools.buildImage {
+        tag = "latest";
+        name = "hyperspace-dali";
+        config = { Entrypoint = [ "${hyperspace-dali}/bin/hyperspace" ]; };
```

Comment:
just doube check it run. oftent there is lack of deps or deps not found.

Reply:
looks like all ok
```
$ docker run --rm -ti hyperspace-dali:latest --help
hyperspace-core 
Possible subcommands of the main binary

USAGE:
    hyperspace <SUBCOMMAND>

OPTIONS:
    -h, --help    Print help information

SUBCOMMANDS:
    create-channel       Creates a channel on the specified port
    create-clients       Creates light clients on both chains
    create-connection    Creates a connection between both chains
    fish                 Start the relayer in fishing mode (catching malicious transactions)
    help                 Print this message or the help of the given subcommand(s)
    relay                Start relaying messages between two chains
```